src/interface/contourplot.jl

```diff
@@ -0,0 +1,72 @@
+"""
+    contourplot(x, y, z; kwargs...)
+
+Description
+============
+
+Draws a contour plot on a new canvas. `levels` controls the number of contour levels.
+
+Usage
+=====
+    contourplot(x, y, z; kwargs...)
+
+Arguments
+=========
+
+- **`levels`** : The number of contour levels. 
+
+- **`colormap`** : The colormap to use for the contour lines. 
+
+- **`colorbar`** : Toggle the colorbar. 
+
+$DOC_PLOT_PARAMS
```

Comment:
Do we need to update `DOC_PLOT_PARAMS` with new item `blend`?

Reply:
Ha yes, good point.